genesis/src/main.rs

```diff
@@ -147,6 +150,14 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                 .required(true)
                 .help("Path to file containing the bootstrap leader's storage keypair"),
         )
+        .arg(
+            Arg::with_name("libra_mint_keypair_file")
+                .long("libra-mint-keypair")
+                .value_name("LIBRA MINT KEYPAIR")
+                .takes_value(true)
+                .required(true)
```

Comment:
Libra keypair probably shouldn’t be required

Reply:
Yes, this is the hack that I brought up on discord.

Reply:
Ah, you mean a required argument. Yea it could be optional then don't add the libra account to genesis.

Reply:
Yeah that :)